packages/web/src/pages/chat-page/components/InboxUnavailableMessage.tsx

```diff
@@ -68,8 +68,8 @@ export const InboxUnavailableMessage = ({
       break
     case ChatPermissionAction.UNBLOCK:
       content = (
-        <>
-          {messages.unblock}{' '}
+        <Text>
+          {messages.unblock}
```

Comment:
I'm not going to die on this hill but adding the space here made more sense to me :joy: